components/fraudlabs_pro/actions/screen-order/screen-order.mjs

```diff
@@ -0,0 +1,272 @@
+import  * as flpnodejs from 'fraudlabspro-nodejs';
+
+export default {
+  name: "Screen Order",
+  description: "Screen an order transaction for payment fraud. This action will detect all possibles fraud traits based on the input parameters supplied. The more input parameter supplied, the higher accuracy of fraud detection. Please refer to the [documentation](https://www.fraudlabspro.com/developer/api/screen-order) for the explanation of the result returned.",
+  key: "fraudlabs_pro-screen-order",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    flp_api_key: {
+      type: "app",
+      app: "fraudlabs_pro",
+      description: "API license key. You can sign up for a trial key at [here](https://www.fraudlabspro.com/subscribe?id=1).",
+    },
+    ip_address: {
```

Comment:
please rename these properties in a lowerCase style in that way we are consistent with rest of  components.
```suggestion
    ip: {
```